qgepplugin/tools/qgepmaptooladdfeature.py

```diff
@@ -75,7 +75,7 @@ def reset3D(self):
 
     def asGeometry3D(self):
         wkt = 'LineStringZ('\
-              + ', '.join(['{} {} {}'.format(p.x(), p.y(), p.z()) for p in self.points])\
+              + ', '.join(['{} {} {}'.format(p.x(), p.y(), 0 if math.isnan(p.z()) else p.z()) for p in self.points])\
```

Comment:
Should this be '/qgis/digitizing/default_z_value' instead?

CC @ponceta

Reply:
Better yes.